2017-01-30-openwhisk-integration-with-serverless

```diff
@@ -0,0 +1,44 @@
+
+## OpenWhisk + The Serverless Framework
+Developers can use a [simple manifest file](https://serverless.com/framework/docs/providers/openwhisk/guide/services#serverlessyml) to define [Serverless functions](https://serverless.com/framework/docs/providers/openwhisk/guide/functions/), connect them to [event sources](https://serverless.com/framework/docs/providers/openwhisk/guide/events/) and declare cloud services needed by their application.
+
+The framework then [deploys these Serverless applications](https://serverless.com/framework/docs/providers/openwhisk/cli-reference/deploy/) to the cloud provider.
+
+### Introducing OpenWhisk Support
+Multi-provider support was a goal we laid out following [the Serverless Framework v1 release](https://serverless.com/blog/serverless-post-1.0/). With the OpenWhisk integration, developers using the framework can choose to deploy their Serverless apps to any OpenWhisk platform instance. Further, multi-provider support simplifies the process of moving applications between cloud providers and enables the development of multi-cloud Serverless apps.
+
+The Serverless workflow and developer experience is consistent across all providers. You don't need to learn custom commands or syntax for each platform.
+
+## Resources for Getting Started
+Check out these resources to help you get started with OpenWhisk integration.
+
+### Intro to the OpenWhisk Serverless Plugin Video
+Learn more about how to use the Serverless Framework with the new OpenWhisk provider plugin in this [quick video].
+
+### Serverless Docs
+OpenWhisk is now included in the [Serverless Docs](https://serverless.com/framework/docs/). You'll find a guide to building Serverless applications, CLI command reference, platform event support and an example application.
+
+### Serverless Examples Repository
+Check out the [Serverless Examples repository](https://github.com/serverless/examples) to see more sample applications. OpenWhisk examples include: how to build HTTP APIs, cron-based schedulers, chaining functions and more.
+
+## Let Us Know What You Think
+For months we've been collaborating with the OpenWhisk team to ensure a great user experience and seamless integration. Let us know what you think. Community feedback is a driving force in the direction of the Serverless Framework.
+
+You can report bugs or request features by opening an issue in the [Serverless-OpenWhisk repository](https://github.com/serverless/serverless-openwhisk). Join the conversation in the [Serverless forum](http://forum.serverless.com/), [chat room](https://gitter.im/serverless/serverless) or this [Slack channel for OpenWhisk](http://slack.openwhisk.org/).
+
+We're excited to hear your feedback.
+
+## What's Next
+The roadmap includes support for non-Node.js runtimes, ensuring compatibility with popular third-party plugins and integrating new features from the platform.
```

Comment:
I would add that this is the Roadmap for the OpenWhisk integration so that users don't think it's the one for Serverless itself.